### PR TITLE
Migrate Overlay + Unit tests + Playground

### PR DESCRIPTION
**Pull request by MoveUpwardsDev**
#50

### CODE REVIEW

File: Sejima/SejimaTests/MUOverlayTests.swift [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        XCTAssertEqual(overlay.borderWidth, 2)
+        XCTAssertEqual(overlay.radius, 0)
+        XCTAssertEqual(overlay.borderColor, .white)
+    }
+
+    func testCustomValues() {
+        let overlay = MUOverlay()
+        XCTAssertNotNil(overlay)
+
+        overlay.style = .custom(8)
```
Comment (damien-nd) [inline Sejima/SejimaTests/MUOverlayTests.swift:28]: .custom(8) will be erase by radius = 4.
Should be nice to test it before or to add more tests in testOverlayRadius.

File: Sejima/Source/MUOverlay/MUOverlay.swift [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+    /// Describes the Overlay's border color appearance while it shows
+    @IBInspectable open dynamic var borderColor: UIColor = .white {
+        didSet {
+            overlay.layer.borderColor = borderColor.cgColor
+        }
+    }
+
+    /// The overlay's style.
+    open var style: MUCornerStyle = .round {
```
Comment (damien-nd) [inline Sejima/Source/MUOverlay/MUOverlay.swift]: As radius has the init value to 0.0, it should be better to have .square here?

File: Sejima/Source/MUOverlay/MUOverlay.swift [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+            guard radius > 0 else {
+                return
+            }
+            style = .custom(radius)
+        }
+    }
+
+    // MARK: - Private functions
+
+    private func updateStyle() {
```
Comment (damien-nd) [inline Sejima/Source/MUOverlay/MUOverlay.swift:63]: If this function is duplicated in other views, should be nice to add an extension of UIView + MUCornerStyle.

File: Sejima/Source/MUOverlay/MUOverlay.swift [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+    // MARK: - Life cycle functions
+
+    /// Default setup to load the view from a xib file.
+    override open func xibSetup() {
+        super.xibSetup()
+
+        layer.masksToBounds = true
+        overlay.layer.masksToBounds = true
+        overlay.layer.borderColor = borderColor.cgColor
```
Comment (damien-nd) [inline Sejima/Source/MUOverlay/MUOverlay.swift]: To remove this 3 lines, can we use Apple default values?

File: Sejima/Source/MUOverlay/MUOverlay.swift (hunk 3 above)
Comment (MoveUpwardsDev) [reply Sejima/Source/MUOverlay/MUOverlay.swift:63]: It's not possible to add properties in extension

**Pull request merged**